bool replan
bool action_done
int32[] pos
int32 my_degree
string mission_name
float32 time
---
int32 degree
int32 speed
int32 mode
int32[] pos
int32[] ST2
int32 is_wait
string mission_name 

